teamsalerts/main.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_lambda_function" "sns_to_teams" {
```

Comment:
Not sure about this, so feel free to ignore, but you may be able to trim this down a bit by using the lambda module in this repo? https://github.com/massgov/mds-terraform-common/blob/1.x/lambda/main.tf Some of this stuff is duplicated - for example, this lambda function, the cloudwatch log group, the policy document...

Here is an example of using it: https://github.com/massgov/mds-terraform-common/blob/1.x/webhook-converters/github-to-teams/main.tf#L33-L60

Again, not sure if it would work or if there's a reason you did it differently, so feel free to ignore

Reply:
yeah I went back and forth on this, but I think you're right. I initially opted to not use that module because it creates a few resources and exposes some configurability that were all pretty unnecessary, but now that I take a look again at the lambda module, there are definitely some advantages to using the abstraction (e.g. I forgot to create an alarm for JS errors, which comes in the box with the module). I'll update!